packages/lex_service/tsconfig.json

```diff
@@ -16,6 +16,7 @@
     "noImplicitAny": true,
     "strictBindCallApply": true,
     "forceConsistentCasingInFileNames": true,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+    "lib": ["es2019", "dom"]
```

Comment:
There shouldn't be any need for changing the tsconfig.

Reply:
changed it back